test_scripts/Defects/5_0/1846_Mismatch_for_mandatory_parameter_between_MOBILE_API_and_HMI_API.lua

```diff
@@ -0,0 +1,56 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/SmartDeviceLink/sdl_core/issues/1846 
+--
+-- Description:
+-- Mismatch for mandatory parameter between MOBILE_API and HMI_API
+--
+-- There is a mismatch for the mandatory parameter between the MOBILE_API and the HMI_API for the 
+-- following parameters:
+--     distanceToManeuver
+--     distanceToManeuverScale
+-- The MOBILE_API is reflecting that these parameters are not mandatory, whereas the HMI_API 
+-- reflects that they are mandatory.
+-- 
+-- WARNING: you should include "Navigation-1" group in "sdl_preloaded_pt.json" file before run 
+-- this test!
+---------------------------------------------------------------------------------------------------
+
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('user_modules/sequences/actions')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Functions ]]
+local function checkShowConstantTBT(pParams, pResultTable)
+	local cid = common.getMobileSession():SendRPC("ShowConstantTBT", pParams)
+	common.getMobileSession():ExpectResponse(cid, pResultTable)
+end
+
+--[[ Scenario ]]
+runner.Title("Preconditions")
+runner.Step("Clean environment", common.preconditions)
+runner.Step("Start SDL, HMI, connect Mobile, start Session", common.start)
+runner.Step("RAI", common.registerApp)
+runner.Step("Activate App", common.activateApp)
+
+runner.Title("Test")
+runner.Step("Negative check mandatory parameter distanceToManeuver", checkShowConstantTBT, {{
+		appID = common.getHMIAppId(1),
```

Comment:
@KrupenichOleg `appID` is not a part of parameters list for `ShowConstantTBT` in `MOBILE` API. Please notice list of parameters for `MOBILE` API could differ from list for `HMI` API

Reply:
@KrupenichOleg The same in lines `45` and `50`